oauth2_http/javatests/com/google/auth/oauth2/MockTokenServerTransport.java

```diff
@@ -16,10 +17,12 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import java.lang.UnsupportedOperationException;
+
 /**
  * Mock transport to simulate providing Google OAuth2 access tokens
  */
-public class MockTokenServerTransport extends MockHttpTransport {
+public class MockTokenServerTransport extends HttpTransport {
```

Comment:
As discussed offline, I'll undo this change since it's not related to the rest of the changes. 

As for your concern, MockHttpTransport already extends HttpTransport, so this is only cutting out one layer of the hierarchy, which happens to not be used at all by MockTokenServerTransport.